File: python/read-file.py
import json
from dateutil import parser

with open('file.json', 'r') as file:
    jsonData = json.load(file)

def parse_timestamp(timestamp):
    return parser.isoparse(timestamp)

for course in jsonData:
    print(f"Curso: {course['title']}")

    for lesson in course['lessons']:
        print(f"  Aula {lesson['id']}: {lesson['title']}")
        print(f"    Media: {lesson['media']}")
        print(f"    Timestamp: {parse_timestamp(lesson['timestamp'])}")

    print()

File: javascript/js/read-file.js
import { jsonData } from "./file.js";

const parseTimestamp = (timestamp) => {
    return new Date(timestamp);
};

for (const course of jsonData) {
    console.log(`Curso: ${course.title}`);

    for (const lesson of course.lessons) {
        console.log(`      Aula ${lesson.id}: ${lesson.title}`);
        console.log(`           Media: ${lesson.media}`);
        console.log(`           Timestamp: ${parseTimestamp(lesson.timestamp)}`);
    }

    console.log();
}